srtcore/buffer.h

```diff
@@ -440,7 +440,7 @@ class CRcvBuffer
        return m_iStartPos == m_iLastAckPos;
    }
    bool full() const { return m_iStartPos == (m_iLastAckPos+1)%m_iSize; }
-   int capacity() const { return m_iSize; }
+   int capacity() const { return m_iSize; } // XXX WHY it was changed to m_iSize-1 ?
 
```

Comment:
> // XXX WHY it was changed to m_iSize-1 ?
This comment was already removed from one of the previous PRs.

Reply:
Ok, I'll remove this. We'll research during the reimplementation